docs/tutorials/onnx2tensorrt.md

```diff
@@ -0,0 +1,78 @@
+# Tutorial 6: ONNX to TensorRT (Experimental)
+
+<!-- TOC -->
+
+- [Tutorial 6: ONNX to TensorRT (Experimental)](#tutorial-6-onnx-to-tensorrt-experimental)
+  - [How to convert models from ONNX to TensorRT](#how-to-convert-models-from-onnx-to-tensorrt)
+    - [Prerequisite](#prerequisite)
+    - [Usage](#usage)
+  - [List of supported models convertable to TensorRT](#list-of-supported-models-convertable-to-tensorrt)
+  - [Reminders](#reminders)
+  - [FAQs](#faqs)
+
+<!-- TOC -->
+
+## How to convert models from ONNX to TensorRT
+
+### Prerequisite
+
+1. Please refer to [get_started.md](https://mmdetection.readthedocs.io/en/latest/get_started.html) for installation of MMCV and MMClassification from source.
+2. Please refer to [ONNXRuntime in mmcv](https://mmcv.readthedocs.io/en/latest/onnxruntime_op.html) and [TensorRT plugin in mmcv](https://github.com/open-mmlab/mmcv/blob/master/docs/tensorrt_plugin.md) to install `mmcv-full` with ONNXRuntime custom ops and TensorRT plugins.
```

Comment:
Check if mmclassification has to use mmcv-full and TensorRT plugins in mmcv.

Reply:
You could remove the 2nd Prerequisite since there are no custom ops used in MMCLS.